### ISSUE TITLE
its not updated and wont update

### ISSUE THREAD

**Issue by SoapCanMan** [CJMinecraft01/ForgeModBuilder#30]
when i launch it it says new update available so i click yes and it says a new update is available so i click yes and replace the file and it says an exception occurred with web client request. also if i put it someplace else if i try to update it it say it has an update over and over and over. i think its trying to update but the update link is just downloading the previous version so it keeps asking to download but cant because the link isn't up to date.

TLDR: update the download link.

**Comment by SoapCanMan** [comment]
i really want to use this but i can't as the download link isn't correct and the 0.1.1.3 is very broken please fix the link so i can easily set up the stuff the forge instructions to do this without the builder just confuse me.

i feel like this might be simple tho i have no idea if it is

**Comment by industrialmaniac314** [comment]
same happens here, can't do anything
